src/components/Tokens/TokenTable/SearchBar.tsx

```diff
@@ -59,10 +59,15 @@ const SearchInput = styled.input`
 `
 
 export default function SearchBar() {
-  const [localFilterString, setLocalFilterString] = useState('')
+  const currentString = useAtomValue(filterStringAtom)
+  const [localFilterString, setLocalFilterString] = useState(currentString)
   const setFilterString = useUpdateAtom(filterStringAtom)
   const debouncedLocalFilterString = useDebounce(localFilterString, 300)
 
+  useEffect(() => {
+    setLocalFilterString(currentString)
+  }, [currentString])
```

Comment:
where else in the app would be updating this value?

Reply:
https://github.com/Uniswap/interface/blob/0b9fda5b25adc56fed0a29ca84eb7d38c62ca348/src/pages/Tokens/index.tsx#L71

Reply:
makes sense 👍